src/osx/cocoa/toolbar.mm

```diff
@@ -1170,160 +1170,162 @@ - (BOOL) isFlipped
 
 bool wxToolBar::Realize()
 {
-    if ( !wxToolBarBase::Realize() )
-        return false;
+    if( dynamic_cast<wxFrame *>( GetParent() )->GetToolBar() )
```

Comment:
This is just wrong. First of all, dynamic cast may fail and the code would just crash.

Second, what's the point of doing it? I have no idea why is this test needed here.